app/sessionStore.js

```diff
@@ -252,6 +252,11 @@ module.exports.cleanAppData = (data, isShutdown) => {
   const clearAutofillData = isShutdown && getSetting(settings.SHUTDOWN_CLEAR_AUTOFILL_DATA) === true
   if (clearAutofillData) {
     autofill.clearAutofillData()
+    const date = new Date().getTime()
```

Comment:
this works, but fyi you can just delete it because it will be-readded from the default session data